x-pack/plugins/infra/public/observability_logs/log_view_state/src/defaults.ts

```diff
@@ -5,4 +5,8 @@
  * 2.0.
  */
 
-export * from './source_id';
+const DEFAULT_LOG_VIEW_ID = 'default';
+export const DEFAULT_LOG_VIEW = {
+  type: 'log-view-reference' as const,
+  logViewId: DEFAULT_LOG_VIEW_ID,
+};
```

Comment:
**question**: this is the same as defined in the higher level [x-pack/plugins/infra/common/constants.ts](https://github.com/elastic/kibana/pull/152933/files#diff-0329fbc8cc20930fa4d3ec0891f4ec0f0ff967a2877ab649af2793ea74da18cb), do we need to explicitly redefine it here for when the observability_logs could be used as a package?

Reply:
I think this was just a leftover of my initial changes vs the final product. I'll change the references to all point to `observability_logs/log_view_state/src/defaults.ts`, that should be our source of truth.